tests/e2e/pkg/tester/kubectl.go

```diff
@@ -0,0 +1,192 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tester
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/kubetest2/pkg/exec"
+)
+
+// AcquireKubectl obtains kubectl and places it in a temporary directory
+func (t *Tester) AcquireKubectl() (string, error) {
+	// first, get the name of the latest release (e.g. v1.20.0-alpha.0)
+	if t.Ginkgo.TestPackageVersion == "" {
+		cmd := exec.Command(
+			"gsutil",
+			"cat",
+			fmt.Sprintf("gs://%s/%s/latest.txt", t.Ginkgo.TestPackageBucket, t.Ginkgo.TestPackageDir),
+		)
+		lines, err := exec.OutputLines(cmd)
+		if err != nil {
+			return "", fmt.Errorf("failed to get latest release name: %s", err)
+		}
+		if len(lines) == 0 {
+			return "", fmt.Errorf("getting latest release name had no output")
+		}
+		t.Ginkgo.TestPackageVersion = lines[0]
+
+		klog.V(1).Infof("Kubectl package version was not specified. Defaulting to latest: %s", t.Ginkgo.TestPackageVersion)
```

Comment:
I think we could log this without V(1) (i.e. at v=0), particularly as it seems like it's still giving us some trouble!